app/src/main/java/org/simple/clinic/di/NetworkModule.kt

```diff
@@ -53,6 +54,7 @@ open class NetworkModule {
         .add(Answer.MoshiTypeAdapter())
         .add(Gender.MoshiTypeAdapter())
         .add(PatientPhoneNumberType.MoshiTypeAdapter())
+        .add(DayOfMonth.MoshiTypeConverter)
```

Comment:
We're using `MoshiTypeAdapter` in the rest of the codebase. Could you please rename this to follow the same convention?

Reply:
done